src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentForm.tag

```diff
@@ -13,32 +13,54 @@
 <%@ attribute name="divId" required="true" %>
 <%@ attribute name="formType" required="true" %>
 <%@ attribute name="textAreaId" required="true" %>
-<%@ attribute name="submitLink" required="true" %>
+<%@ attribute name="submitLink" %>
 <%@ attribute name="buttonText" required="true" %>
 <%@ attribute name="viewType" %>
 <%@ attribute name="isOnQuestionsPage" %>
+<%@ attribute name="moderatedPersonEmail" %>
+<%@ attribute name="isModeration" %>
+
 <c:set var="isEditForm" value="${formType eq 'Edit'}" />
 <c:set var="isAddForm" value="${formType eq 'Add'}" />
-<form class="responseComment${formType}Form"<c:if test="${isEditForm}"> style="display: none;" id="responseCommentEditForm-${divId}"</c:if>>
-  <div class="form-group form-inline">
-    <div class="form-group text-muted">
-      <p>
-        Giver: ${fn:escapeXml(frc.responseGiverName)}
-        <br>
-        Recipient: ${fn:escapeXml(frc.responseRecipientName)}
-      </p>
-      You may change comment's visibility using the visibility options on the right hand side.
+<c:choose>
+  <c:when test="${frc.commentFromFeedbackParticipant}">
```

Comment:
Can we extract the code for student side to a new tag.

I see the changes is kind of big and may not be a good choice to reuse the existing tag.

Reply:
Yes, that's better.